tests/live/test_workflow.py

```diff
@@ -95,8 +95,12 @@ def test_iac_workflow__happy_path(pyiac):
 
     info("Checking instance creation")
     instances = iac("instance list")
+    info(instances)
     assert instance_name in {i["name"] for i in instances}
 
+    info("Giving the system some time to startup")
```

Comment:
We sometimes get an error when we copy the file to the remote system that the connection is refused.  I think that may be caused by the EC2 instance running, but the system has not yet started.  Adding a sleep to give the system some time to startup.